minishift-ci-index.yaml

```diff
@@ -39,7 +39,7 @@
     name: '{git_repo}'
     description: |
         {jobdescription}
-    node: '{ci_project}'
+    node: 'minishift'
```

Comment:
@bstinsonmhk Could you make this change here https://github.com/minishift/minishift-ci-jobs/blob/master/minishift-ci-index.yaml#L73 ? It will one for all. 

Thanks for all effort :+1:  :100: